src/main/java/net/fabricmc/fabric/mixin/commands/MixinServerCommandManager.java

```diff
@@ -37,8 +37,9 @@
 	@Inject(method = "<init>(Z)V", at = @At("RETURN"))
 	public void addMethods(boolean dedicated, CallbackInfo info) {
 		// TODO: Run before findAmbiguities
-		if(!dedicated){
```

Comment:
Pretty sure the code is this way as we hook DedicatedServers elsewhere

Reply:
They didn't work when I tried it, see #142.

Reply:
This commit should have fixed it: https://github.com/FabricMC/fabric/commit/98c307909e319e174c34badd96630419abd57b59